codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompTestConstants.java

```diff
@@ -18,8 +18,8 @@
 public final class StompTestConstants {
     public static final String CONNECT_FRAME =
         "CONNECT\n" +
-            "host:stomp.github.org\n" +
             "accept-version:1.1,1.2\n" +
+            "host:stomp.github.org\n" +
```

Comment:
@Scottmitch the others being stomp and spyd? Well they don't need it either :-). If there is a protocol that needs it, just plug in a LinkedHashMap.